### ISSUE TITLE
HoneySQL 2.0

### ISSUE THREAD

**Issue by seancorfield** [nilenso/honeysql-postgres#62]
I just released 2.0.0-alpha1 of HoneySQL which:
* uses a different extension mechanism
* includes all of nilenso/honeysql-postgres functionality (with some small differences)

https://github.com/seancorfield/honeysql/tree/v2 -- `seancorfield/honeysql {:mvn/version "2.0.0-alpha1"}`

I would very much like folks who use this library to test HoneySQL 2.0 and provide feedback on the PostgreSQL support (and the small changes I've had to make). In particular, I've copied the tests from this library into HoneySQL 2.0 and made some (what I hope are minimal) changes: https://github.com/seancorfield/honeysql/blob/v2/test/honey/sql/postgres_test.cljc

If I've missed anything from this library -- or you really don't like the changes I've made in syntax -- please open issues on the HoneySQL repo so I can address them during the prerelease phase for 2.0.

Thank you for all the work you've done on PostgreSQL support for HoneySQL!

**Comment by scimetfoo** [comment]
That's exciting @seancorfield! I'd explored 2.0 a little bit a month or so ago to get a general sense of its direction. I'll take a thorough look at it this week.

**Comment by scimetfoo** [comment]
That's exciting @seancorfield! I'd explored 2.0 a little bit a month or so ago to get a general sense of its direction. I'll take a thorough look at it.

**Comment by seancorfield** [comment]
FYI: HoneySQL 2.0.0 Beta 2 has parity with 0.4.112, albeit with a few syntax changes. I saw the update here that you plan to continue maintaining `honeysql-postgres` so I wanted to check if there is anything specific you'd like me to add to the HoneySQL docs around that? Right now, I have a few cautions in the v2 docs that `honeysql-postgres` doesn't work with v2 (only with v1), but fairly soon I'll be merging the v2 branch back to develop (and creating a v1 maintenance branch) so v2 will be the default version.

**Comment by seancorfield** [comment]
Since HoneySQL 2.0 is "gold" now, I'm going to close this. I'll continue to keep an eye on this repo and add to HoneySQL anything that gets added here, going forward.

**Comment by kitallis** [comment]
Thanks for letting us know @seancorfield. None of us are currently actively using the honeysql-space (either this extension or core) so haven't been following up on the progress of 2.0.

I'll add a note about HoneySQL 2.0 reaching "gold" in the README. 

For now, our idea was simply to maintain the existing feature set for people who are on 1.x + this extension, and accept any new PRs and mostly not add new things by ourselves (unless there's open issues). We'll try to redirect new-comers who want a feature to HoneySQL 2.0 if they don't mind using the new library.

In other words, honeysql-postgres doesn't really have a feature roadmap from our end. We'll try to redirect people as necessary.
